### PR TITLE
Protect runtimehost with AAD

### PR DESCRIPTION
**Pull request by rekhaswaminathan**
- Runtimehost is protected with aad to allow request only from applens
- Healthping controller has not changed so that we can test it to see if the app is up
- In the future sprint, runtimehost will be protected to allow requests from partner app ids as well.

### CODE REVIEW

File: src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs [hunk 1]
```diff
@@ -8,9 +8,11 @@
 using Diagnostics.RuntimeHost.Models;
 using Diagnostics.RuntimeHost.Services;
 using Diagnostics.RuntimeHost.Utilities;
+using Microsoft.AspNetCore.Authorization;
 
 namespace Diagnostics.RuntimeHost.Controllers
 {
+    [Authorize]
     public abstract class SiteControllerBase : DiagnosticControllerBase<App>
```
Comment (ShekharGupta1988) [inline src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs]: We add the tag to Abstract base class as well? Will this work?

File: src/Diagnostics.RuntimeHost/appsettings.json [hunk 2]
```diff
@@ -86,5 +86,10 @@
   },
   "SearchAPI": {
     "SearchAPIEnabled": false
+  },
+  "AzureAD": {
+    "AADAuthority": "https://login.microsoftonline.com/microsoft.onmicrosoft.com",
+    "ClientId": "b9a1efcd-32ee-4330-834c-c04eb00f4b33",
```
Comment (ShekharGupta1988) [inline src/Diagnostics.RuntimeHost/appsettings.json]: Lets keep app settings clean.

File: src/Diagnostics.RuntimeHost/appsettings.json [hunk 3]
```diff
@@ -86,5 +86,10 @@
   },
   "SearchAPI": {
     "SearchAPIEnabled": false
+  },
+  "AzureAD": {
```
Comment (ShekharGupta1988) [inline src/Diagnostics.RuntimeHost/appsettings.json]: how about renaming this to something more apt like 'AccessControlSettings' or 'SecuritySettings', something like that.

File: src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs (hunk 1 above)
Comment (hforeste) [reply src/Diagnostics.RuntimeHost/Controllers/Site/SiteControllerBase.cs]: You also have authorize attribute on the `SitesController` class so I assume you won't need this here.

**Pull request merged**